idaes/core/util/tests/test_model_diagnostics.py

```diff
@@ -82,6 +1261,22 @@ def v_exp():
     ).T
 
 
+@pytest.mark.unit
+def test_deprecate_degeneracy_hunter(caplog):
+    m = ConcreteModel()
+    m.v = Var()
+    m.o = Objective(expr=m.v)
+    dh = DegeneracyHunter(m)
+
+    msg = (
+        "DEPRECATED: DegeneracyHunter is being deprecated in favor of the new "
+        "DiagnosticsToolbox.  (deprecated in 2.0.0, will be removed in (or after) 3.0.0)"
```

Comment:
```suggestion
        "DiagnosticsToolbox.  (deprecated in 2.2.0, will be removed in (or after) 3.0.0)"
```

I think this is what is currently causing the tests to fail.